docs/_articles/en/lwc/write-lwc.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Write Lightning Web Components in VS Code
+lang: en
+---
+
+The Lightning Web Components Extension for VS Code leverages the editor's built-in code navigation and language features so that you can efficiently build Lightning web components. 
+
+Take advantage of the following Lightning Web Components Extension for VS Code features:
+- [Code Completion](#code-completion)
+- [View Documentation on Hover](#view-documentation-on-hover)
+- [View Errors Messages on Hover](#view-error-messages-on-hover)
+- [Linting](#linting)
+
+## Code Completion
+
+The Lightning Web Components Extension builds on VS Code's code completion capabilities by completing resource names in HTML and JavaScript files.
+
+VS Code completes static resources. In this example, it completes the `chartjs` object in the `libChartjs` component. `libChartjs` refers to the third-party JavaScript library [chartjs](https://www.chartjs.org/). Include it in your code by importing `loadScript` from the Lightning Web Components `platformResourceLoader` module. Then, add the statement `import chartjs from '@salesforce/resourceUrl/chart';`. Here, VS Code fills in the resource's name when we add it in the import statement.
+
+![Static Resource Completion](./images/vscode_lwc_staticresource.png)
+
+VS Code also completes Lightning web components imports from the `lwc` module. Here, it completes `LightningElement`. When hovered over, `LightningElement` displays information about the `LightningElement` class. 
```

Comment:
This is probably where we can mention other imports like @wire @api @track, etc

Reply:
Could we just say something like: VS Code completes Lightning web components imports from the `lwc` module, as well the Lightning Web Components framework's reactive properties and wire service (add links to the LWC dev guide for both). Does this also apply for Salesforce modules?

Reply:
Yeah that sounds fine